datumaro/plugins/camvid_format.py

```diff
@@ -208,6 +216,25 @@ def _lazy_extract_mask(mask, c):
 
 
 class CamvidImporter(Importer):
+    @classmethod
+    def detect(cls, context: FormatDetectionContext) -> None:
+        annot_path = context.require_file('*.txt',
+            exclude_fnames=CamvidPath.LABELMAP_FILE)
+
+        with context.probe_text_file(
+            annot_path, 'must be a CamVid-like annotation file',
+        ) as f:
+            for line in f:
+                _, mask_path = _parse_annotation_line(line)
```

Comment:
The function throws an exception. I think it should be catched here.

Reply:
Don't forget, any thrown exception within `probe_text_file` results in the requirement being considered unmet - which is exactly what we want in that case. 🙂